bundles/org.openhab.binding.ecobee/src/main/java/org/openhab/binding/ecobee/internal/api/EcobeeApi.java

```diff
@@ -319,7 +319,7 @@ private void logIOException(Exception e) {
                     rootCause.getMessage());
         } else {
             // What's left are unexpected errors that should be logged as INFO with a full stack trace
```

Comment:
The comment says they should be logged as INFO but now they are logged as DEBUG

Reply:
Good catch, I've corrected it.